src/pages/billing/Project.js

```diff
@@ -645,7 +651,7 @@ const ProjectDetail = ({ authorizeAndLoadAccounts, billingAccounts, billingProje
       }
     }),
     billingAccountsOutOfDate && h(BillingAccountSummaryPanel, { counts: _.mapValues(_.size, groups) }),
-    updating && spinnerOverlay
+    updating && absoluteSpinnerOverlay
```

Comment:
This causes the spinner to render in the same location as the one for the parent page.

Reply:
I wonder if we should note this as a comment above the definition of `absoluteSpinnerOverlay`.

Reply:
My only issue with this is that it would prevent selecting another project if your spend data is taking a long time to load.